mkdocs/__main__.py

```diff
@@ -99,9 +100,10 @@ def cli():
 @click.option('-a', '--dev-addr', help=dev_addr_help, metavar='<IP:PORT>')
 @click.option('-s', '--strict', is_flag=True, help=strict_help)
 @click.option('-t', '--theme', type=click.Choice(theme_choices), help=theme_help)
+@click.option('-e', '--theme-dir', type=click.Path(), help=theme_dir_help)
```

Comment:
`-d` is already used for `--site-dir`. I didn't want to have different options between `serve` and `build`.